Add paella player 8 docs (#7267)
opencast/opencast: docs/guides/admin/docs/configuration/player/paella.player8/plugins.md

Paella player plugins
=====================

Almost every paella feature is a plugin that can be enabled/disabled by each organization.
You need to modify the `plugins` section of the [paella config file](configuration.md).

> To enable/disable a plugin you need to set the plugin `enable` property to `true`/`false`.

Reviewer: Example below uses `enabled`, here we say `enable`.  Hopefully it's all either `enable`, or `enabled`, and not a mixed case :)

```suggestion
To enable/disable a plugin you need to set the plugin `enabled` property to `true`/`false`.
```

Example:

> ```

Reviewer 2: ```suggestion
```json
```

[org.opencast.paella.editorPlugin](https://polimediaupv.github.io/paella-opencast/reference/paella-opencast-plugins/plugin_editorplugin/)                                       | Adds a button in the player interface that allows access to the Opencast video editor to modify and enhance video content.
[org.opencast.paella.eventDetailsPlugin](https://polimediaupv.github.io/paella-opencast/reference/paella-opencast-plugins/plugin_eventdetailsplugin/)    | Adds a panel with the video description information.
[org.opencast.paella.loginPlugin](https://polimediaupv.github.io/paella-opencast/reference/paella-opencast-plugins/plugin_loginplugin/)                                         | Adds a login button in the player interface that allows users to authenticate and access additional features and personalized content.

> [org.opencast.paella.matomo.userTrackingDataPlugin](https://polimediaupv.github.io/paella-opencast/reference/paella-opencast-plugins/plugin_matomo_usertrackingdataplugin/)     | Allows to use Matomo service to track usage data.

Reviewer: ```suggestion
[org.opencast.paella.matomo.userTrackingDataPlugin](https://polimediaupv.github.io/paella-opencast/reference/paella-opencast-plugins/plugin_matomo_usertrackingdataplugin/)     | Enables user statistics gathering via [a Matomo service](../../../configuration/admin-ui/statistics/matomo-statistics-provider/) to track usage data.
```
